js/src/components/google-ads-account-card/connect-ads/index.js

```diff
@@ -18,15 +18,19 @@ import Subsection from '.~/wcdl/subsection';
 import useApiFetchCallback from '.~/hooks/useApiFetchCallback';
 import useDispatchCoreNotices from '.~/hooks/useDispatchCoreNotices';
 import useGoogleAdsAccount from '.~/hooks/useGoogleAdsAccount';
+import useEventPropertiesFilter from '.~/hooks/useEventPropertiesFilter';
 import AdsAccountSelectControl from './ads-account-select-control';
 import { useAppDispatch } from '.~/data';
+import { FILTER_ONBOARDING } from '.~/utils/tracks';
 import './index.scss';
 
 /**
  * Clicking on the button to connect an existing Google Ads account.
  *
  * @event gla_ads_account_connect_button_click
  * @property {number} id The account ID to be connected.
+ * @property {string} [context] Indicates the place where the button is located.
+ * @property {string} [step] Indicates the step in the onboarding process.
```

Comment:
Do you think we should also add `gla_mc_id` and `gla_version` props in the JSDoc so the [tracking document](https://github.com/woocommerce/google-listings-and-ads/blob/develop/src/Tracking/README.md) could be more accurate?

Reply:
The current `doc:tracking` script (`woocommerce-grow-jsdoc`) doesn't support global event properties, and I'm not leaning toward duplicating global properties for the docs of all frontend tracking events.

It needs more discussion about how the `woocommerce-grow-jsdoc` tool should handle global event properties. I've first listed the global event properties directly at the beginning of the tracking README file. https://github.com/woocommerce/google-listings-and-ads/pull/2421/commits/0e6b33dd25faac87265467958a9cb8367ca4e09b

Reply:
Got it, thanks for clarifying and adding the global event props in the tracking README. 👍